prac_06/languages.py

```diff
@@ -6,8 +6,15 @@ def main():
     python = ProgrammingLanguage("Python", "Dynamic", True, 1991)
     visual_basic = ProgrammingLanguage("Visual Basic", "Static", False, 1991)
 
+    languages = [ruby, python, visual_basic]
 
     print(ruby)
 
+    print("The dynamic languages are:")
+    for i in languages:
```

Comment:
Can use language (singular) for list which is named with the plural (ie. "for language in languages"). Functions exactly correct otherwise.